ebpf/aya-ebpf/src/programs/xdp.rs

```
@@ -32,6 +36,42 @@ impl XdpContext {
     pub fn metadata_end(&self) -> usize {
         self.data()
     }
+
+    /// Adjusts the head of the Packet by given 'delta' (both positive and negative values are
+    /// possible.)
+    #[inline(always)]
+    pub fn adjust_head(&mut self, delta: crate::cty::c_int) -> Result<(), ()> {
+        unsafe {
+            match bpf_xdp_adjust_head(self.ctx, delta) {
+                0 => Ok(()),
+                _ => Err(()),
```

why not return the error?